styles/style.css

```diff
@@ -0,0 +1,235 @@
+section {
+  background-color: #eeeeee;
+  border: 1px solid white;
+  padding: 2em 2em;
+}
+
+body {
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+h2 {
+  font-weight: bold;
+  text-align: center;
+  background-color: #355c7d;
+  color: white;
+  font-family: 'Playfair Display', serif;
+}
+
+/* navigation styles */
+nav {
+  background-color: #ffffff;
+  height: 1em;
+  margin: 1%;
+  font-size: 1.2em;
+  color: #355c7d;
+}
+
+nav ul {
+  display: grid;
+  grid-template-columns: 70% 15% 15%;
+}
+
+nav ul :first-child {
+  grid-column: 1 / 2;
+  color: #f67280;
+}
+
+nav ul :nth-child(2) {
+  grid-column: 2 / 3;
+  text-align: right;
+}
+
+nav ul :nth-child(3) {
+  grid-column: 3 / 4;
+  text-align: center;
+}
+
+/* header styles */
+.header {
+  padding: 2em;
+  background-image: url("../images/header_image.jpg");
+  background-size: cover;
+  background-position: top;
+  text-align: center;
+  background-color: #355c7d;
+  height: 30em;
+}
+
+.header h1 {
+  font-size: 100px;
+  color: #ffffff;
+  margin-top: 1em;
+  margin-bottom: 0em;
+  font-family: 'Playfair Display', serif;
+}
+
+.header sub {
+  font-size: 1.4em;
+  line-height: 2em;
+  color: #ffffff;
+  vertical-align: text-top;
+}
+
+.button {
+  margin: 2em 1em;
+	background-color:#e4685d;
+	border-radius:4px;
+	display:inline-block;
+	cursor:pointer;
+	color:#ffffff;
+	font-size:15px;
+	padding:12px 47px;
+	text-decoration:none;
```

Comment:
Hit all of the code requirements, good work! It looks a lot like the wireframe, but has some good additions that make it pop.